config/adopters.json

```diff
@@ -565,6 +565,15 @@
 				"iot.cyclonedds"
 			]
 		},
+		{
+			"name": "Orange",
+			"homepage_url": "https://www.orange.com/",
+			"logo": "logo-orange.png",
+			"logo_white": "logo-open-orange-white.png",
```

Comment:
`logo_white` value does not match file name.

I guess this is a typo and `logo-open-orange-white.png` should be replaced by `logo-orange-white.png`

Reply:
Good catch! Thanks!